x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/collector/cluster/ClusterStatsMonitoringDocTests.java

```diff
@@ -313,7 +313,7 @@ public void testToXContent() throws IOException {
         final List<XPackFeatureSet.Usage> usageList = singletonList(new MonitoringFeatureSetUsage(false, null));
 
         final NodeInfo mockNodeInfo = mock(NodeInfo.class);
-        Version mockNodeVersion = Version.CURRENT.minimumCompatibilityVersion();
+        var mockNodeVersion = Build.current().version();
```

Comment:
Shouldn't this be `Build.current().minWireCompatVersion()`?

Reply:
(also appears unrelated to the PR title?)

Reply:
It should not matter, it is just a placeholder for a version, but you are right, better not introducing any difference

Reply:
> (also appears unrelated to the PR title?)

Well, I need to change the mocked version from `Version` to `String`, it seemed silly to change it to `Version.CURRENT.minimumCompatibilityVersion().toString()`. 
I can replace it with any string - I will replace it with `randomAlphaOfLengthBetween(6, 32)` to make it more explicit that it is "any string you can use for a version".